src/PIL/_version.py

```diff
@@ -1,2 +1,2 @@
 # Master version for Pillow
-__version__ = "8.3.2"
+__version__ = "8.3.99"
```

Comment:
Worth adding a comment about that version being fictional and used temporarily as a workaround required to have a pinned dependency for pip3.

Reply:
what about using a post release version? 
```suggestion
__version__ = "8.3.2.post1"
```
https://peps.python.org/pep-0440/#public-version-identifiers

Reply:
At first I thought of using a post release version, but I still think is better to have this _odd_ number to make it very clear it isn't an official release. Anyhow I'm open to change it if you think is a better choice to use a post release version.